server/pbench/bin/index-pbench

```diff
@@ -24,7 +27,7 @@ try:
 except ImportError:
     from elasticsearch import VERSION as es_VERSION, Elasticsearch
 
-from pbench import tstos, es_index, es_put_template
```

Comment:
Is there a reason why need to make this change now?  If so, can we do it in a separate PR from this one?

Reply:
Sure.